### PR TITLE
Chore: change release docs

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

# Scope of work
Add releases docs. 

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/docs/.vuepress/config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = {
     editLinks: true,
     editLinkText: 'Help us improve this page ❤️',
     nav: [
-      { text: `v${version}`, link:'/migration-guide' },
+      { text: `v${version}`, link:`/introduction/releases/v${version}` },
```
Comment (justyna-13) [inline packages/vue/docs/.vuepress/config.js:17]: releases isn't in introduction :) I think you meant ```/releases/v${version}```

File: packages/vue/docs/releases/v0.9.0.md [hunk 2]
```diff
@@ -1,15 +1,6 @@
-# Releases
+# version 0.9.0
```
Comment (justyna-13) [inline packages/vue/docs/releases/v0.9.0.md:1]: version in the tables on this page should also be changed from 0.9.1 to 0.9.0 :)

File: packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ storiesOf("Organisms|Header", module)
       activeIcon: {
         default: select(
           "activeIcon",
-          ["", "account", "wishlist", "cart"],
+          ["", "account", "wishlist", "cart", "shipping_test"],
```
Comment (justyna-13) [inline packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js:55]: ```shipping_test``` ?

File: packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js [hunk 4]
```diff
@@ -119,9 +119,9 @@ storiesOf("Organisms|Header", module)
           :search-placeholder="searchPlaceholder"
           :search-value="searchValue"
           :cart-icon="cartIcon"
-          :wishlist-icon="wishlistIcon"
+          :wishlist-icon="shipping_test"
```
Comment (justyna-13) [inline packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js:122]: same as above - ```shipping_test``` ?

**Review by justyna-13 — CHANGES_REQUESTED** [review]
Just some small fixes :+1:

File: packages/vue/docs/.vuepress/config.js (hunk 1 above)
Comment (AdamPawlinski) [reply packages/vue/docs/.vuepress/config.js:17]: yes, thanks.

File: packages/vue/docs/releases/v0.9.0.md (hunk 2 above)
Comment (AdamPawlinski) [reply packages/vue/docs/releases/v0.9.0.md:1]: fixed

File: packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js (hunk 3 above)
Comment (AdamPawlinski) [reply packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js:55]: It shouldn't be here obviously :)

File: packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js (hunk 4 above)
Comment (AdamPawlinski) [reply packages/vue/src/components/organisms/SfHeader/SfHeader.stories.js:122]: like above

**Review by justyna-13 — APPROVED** [review]

**Pull request merged**